DataConnectors/VMware ESXi/Connector_Syslog_VMwareESXi.json

```diff
@@ -0,0 +1,123 @@
+{ 
+    "id": "VMwareESXi",
+    "title": "VMWare ESXi", 
+    "publisher": "VMWare", 
+    "descriptionMarkdown": "The [VMWare ESXi](https://www.vmware.com/products/esxi-and-esx.html) connector allows you to easily connect your VMWare ESXi logs with Azure Sentinel This gives you more insight into your organization's ESXi servers and improves your security operation capabilities.", 
+    "additionalRequirementBanner":"These queries and workbooks are dependent on a parser based on a Kusto Function to work as expected. Follow the steps to use this Kusto functions alias **VMWareESXi** in queries and workbooks. [Follow these steps to get this Kusto functions.](https://aka.ms/sentinelgithubparsersvmwareesxi)",
+    "graphQueries": [ 
+     { 
+         "metricName": "Total data received", 
+         "legend": "VMwareESXi", 
+         "baseQuery": "VMwareESXi" 
+     } 
+    ], 
+          "sampleQueries": [ 
+           { 
+               "description" : "Total Events by Log Type", 
+               "query": "VMwareESXi \n| summarize count() by ProcessName"    
+           },
+        {
+            "description" : "Top 10 ESXi Hosts Generating Events",
+            "query": "VMwareESXi /n | summarize count() by HostName /n | top 10 by count_"  
```

Comment:
\n is correct for character, not  /n. Please fix.

Reply:
corrections have been made